prime-router/src/main/kotlin/azure/ReportFunction.kt

```diff
@@ -260,6 +221,7 @@ class ReportFunction(
             .header(HttpHeaders.CONTENT_TYPE, "application/json")
             .body(
                 JacksonMapperUtilities.allowUnknownsMapper
+                    .writerWithDefaultPrettyPrinter()
```

Comment:
Remove this line.  Not going to break anything, but was removed in a previous PR.

Reply:
Removed.